### ISSUE TITLE
Support for multiple disjoint selections doesn't work anymore in version 3.0.0-beta.10 (worked in beta9)

### ISSUE THREAD

**Issue by farid67** [Simonwep/selection#138]
#### What is the problem?

Multiple disjoint selection doesn't work in the version 3.0.0-beta.10, while it was working in the beta 9. Worth noting that I had to update because I was affected by the bug with the select function (see https://github.com/Simonwep/selection/issues/131)

#### What is the current behavior?
<-- What happens in your situation -->
Multiple disjoint selection keep selecting everything starting from the first selected element, as mentioned here (https://github.com/Simonwep/selection/issues/126). Worth noting that the bug was fixed in the 3.0.0-beta9 version.

#### What is the expected behavior?

Multiple disjoint selections to work as earlier, without selecting element between the different selections.

#### Your environment:
```
Toolset (e.g. webpack, vite, vue-cli...): 
Version (@viselect/<insert package>): @viselect/vanilla@3.0.0-beta.10
Browser:  Firefox
OS:  Ubuntu 20.04
```

**Comment by Simonwep** [comment]
Just tried this on the demo page... it seems to be working?

![LluN67NUSQ](https://user-images.githubusercontent.com/30767528/133466343-be7863cc-c3cb-4ee3-bf48-a05be12986f8.gif)

**Comment by farid67** [comment]
Sorry I didn't explained it perfectly, I was talking about disjoint selections using the Shift Key, exactly as in the https://github.com/Simonwep/selection/issues/126 issue.
Considering we have 8 items, 
Click on the first item + shift then click on the 3rd item, it will select items 1,2 and 3
Then type Ctrl + click on the 6th item, then click + Shift to the 8th, it is supposed to select items 6,7 and 8, but it selects items 1 to 8.

**Comment by farid67** [comment]
I tried using the demo page and it works using the original main.ts file 
However, in my case, I'm using js and i'm importing the script from cdn using the following link
`<script src="https://cdn.jsdelivr.net/npm/@viselect/vanilla@3.0.0-beta.10/lib/viselect.cjs.min.js"></script>`
Where I change the version manually.

Therefore, I tried on the demo page by removing the `import SelectionArea from "@viselect/vanilla";` from the ts file, and only keeping the lines used to create the container and the div items.
I created a js file, and I imported the file as explained above, I created the selection object and I was able to reproduce the bug, where disjoint selections work in version 3.0.0-beta.9 but not anymore in the beta.10.


  Here is the link to the bugged version (https://codesandbox.io/s/ueq7z?file=/index.html)
Here is the link to the same code that works only by switching the version beta10 to beta9 (https://codesandbox.io/s/4t59m?file=/index.html)

Hope it helps!
